### ISSUE TITLE
on web, I get the message "Not implemented". localhost is fine.

### ISSUE THREAD

**Issue by JCLaHoot** [webbukkit/dynmap#2144]
I have a home server setup on a mac, and though everything works fine locally, I can't get it to work via the web.

serverip:8123 just leads to a "Not implemented" message. The good news is that it's better than the connection refused error I was getting before I set up port forwarding, but I can't figure out why it's not rendering the map when it's working locally. Has anyone else had this issue?

**Comment by smmmadden** [comment]
you have to enable the port in your firewall/router to be accessible over the internet.  Just like you did for the server port itself.

**Comment by JCLaHoot** [comment]
yes, I did that. I got a "connection refused" type error before I did that.

**Comment by jwshields** [comment]
It works by localhost;
what about your LAN IP, not WAN? (ex: 192.168.1.25)
This doesn't seem like a problem with ports;

"Not Implemented" is an http response code.
It means you're getting to the dynmap httpd (Jetty) - and your browser is requesting _something_ improperly, and Jetty is saying "No"

What browser are you using?
Are you attempting to hit this over https at all?
Do you have any addons in your browser which cause your headers or information sent to be changed?

Tldr, this doesn't seem to be a dynmap issue based off an initial look.
Do you get any error logs to the console for minecraft when you hit it?

**Comment by JCLaHoot** [comment]
Ok... this makes very little sense, but when I tried accessing via the LAN IP on my server mac, it worked perfectly. 
Then, AFTER doing that, I can access it normally via serverip:8123 on any other device. 
This doesn't sound normal... but at least it works now!
Thanks for the help :)
